### PR TITLE
add SahityaRoy

### LINKED ISSUE #621: SahityaRoy
[Github profile](https://github.com/SahityaRoy)

### PR DESCRIPTION
**Pull request by SahityaRoy**
resolve #621

### CODE REVIEW

**Review by AvidCoder101 — CHANGES_REQUESTED** [review]
@SahityaRoy Just one change, could you please add an image of your Github profile README.md into the `imageUrl`? I'll also let the admins do their reviewing. Thanks (:

**Comment by SahityaRoy** [comment]
Hey i don't know how to add screenshot on issue section

**Review by schmelto — CHANGES_REQUESTED** [review]

**Comment by schmelto** [comment]
> Hey i don't know how to add screenshot on issue section

Just make a screenshot of your github profile and paste it into the description field of the issue (#621)

**Comment by SahityaRoy** [comment]
29 07 2021_14 26 39_REC

**Comment by SahityaRoy** [comment]
done

File: data.json [hunk 1]
```diff
@@ -1192,5 +1192,11 @@
     "githubUsername": "rajivranjanmars",
     "imageUrl": "https://user-images.githubusercontent.com/67179338/126766490-97cc500e-eed4-46d0-959f-799b1d7f1e7c.png",
     "issueNumber": 619
+  },
+  {
+    "name": "Sahitya Roy",
+    "githubUsername": "SahityaRoy",
+    "imageUrl": "https://media-exp1.licdn.com/dms/image/C5603AQEYBGYcnAE7Ug/profile-displayphoto-shrink_400_400/0/1624942139886?e=1632960000&v=beta&t=ZWCRX9YfHKDGkLQVDWYkYsnWqnfbdO2r_-tCN_hb9BA",
```
Comment (schmelto) [inline data.json:1199]:
```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/72821604/127463309-7f3dc638-b2fd-4237-ab93-744a48ceb13c.png",
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Pull request closed without merging**